src/sidebar/components/Annotation/AnnotationShareControl.tsx

```diff
@@ -127,70 +135,70 @@ function AnnotationShareControl({
   return (
     <div className="relative" ref={shareRef}>
       <IconButton
-        icon="share"
+        icon={ShareIcon}
         title="Share"
         onClick={toggleSharePanel}
         expanded={isOpen}
       />
       {isOpen && (
-        <Card
-          classes={classnames(
-            // Prefer width 96 (24rem) but ensure that component isn't wider
-            // than 85vw
-            'w-96 max-w-[85vw]',
-            // Position this Card above its IconButton. Account for larger
-            // IconButtons in touch interfaces
-            'absolute bottom-8 right-1 touch:bottom-touch-minimum',
-            'space-y-2 p-2',
-            // Cards do not have a border in the clean theme. Turn it back on.
-            'theme-clean:border theme-clean:border-solid theme-clean:border-grey-3'
-          )}
+        <div
+          // Position this Card above its IconButton. Account for larger
+          // IconButtons in touch interfaces
+          className="absolute bottom-8 right-1 touch:bottom-touch-minimum"
         >
-          <div className="flex items-center">
+          <Card
+            classes={classnames(
+              // Prefer width 96 (24rem) but ensure that component isn't wider
+              // than 85vw
+              'w-96 max-w-[85vw]',
+              'space-y-2 p-2'
```

Comment:
I'm curious, as I've seen this pattern in many components.

What's the benefit of using `classnames` when all the classes are hardcoded (meaning, they will always evaluate as *truthy* and therefore, be rendered) as opposed to `classes="w-96 max-w-[85vw] space-y-2 p-2"`.

Reply:
It's an ergonomic affordance, not functional.  Using `classnames` makes it easier to add comments and organize classes into conceptual groups, like here, where there are a set of classes that pertain to positioning and then a set that pertain to spacing/padding. See the classes on the enclosing `div` for a better example of more complex sets of classes and comments.

Reply:
I imagined the reason was going in that line. Thanks!